.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
```

Comment:
It looks like the current revision is v4.1.0.

If we don't specify `rev` at all, what version gets loaded?  What happens if an upstream release happens since initial installation?  I'm trying to think of what kind of monitoring we might get from Dependabot.

Reply:
Removing the revision results in a fatal error:
```shell
An error has occurred: InvalidConfigError: 
==> File .pre-commit-config.yaml
==> At Config()
==> At key: repos
==> At Repository(repo='https://github.com/pre-commit/pre-commit-hooks')
=====> Missing required key: rev
Check the log at /Users/kchason/.cache/pre-commit/pre-commit.log
```

With regards to upstream changes, these are pretty simple rulesets, but it looks like dependabot doesn't support `pre-commit` hook updates ([dependabot-core#1524](https://github.com/dependabot/dependabot-core/issues/1524))

 We'll need to manually bump it or use one of the workarounds identified, but I'd suggest we revisit that at a later point if/once we adopt `pre-commit` across (some of) the projects.

Reply:
Ok.  Thank you for checking.

Reply:
Ah, interesting.  There's an autoupdate verb.

https://pre-commit.com/#pre-commit-autoupdate

I'm tempted to add that to the CI, but I'm comfortable waiting.